### DISCUSSION TITLE
auto_reload_on_write property of nvim-tree is not reloading after moving files

### DISCUSSION THREAD

**Discussion by imtheaman** [NvChad/NvChad#1157] [Q&A] [answered]
if there's any key bindings for moving files and creating files please reply me. i counldn't find any on the website.
*i haven't added anything in the init.lua. checked the init.lua, it has already included default config of nvim-tree.

**Comment by siduck** [comment, marked as answer]
press g? in nvimtree , these are the default ones

![image](https://user-images.githubusercontent.com/59060246/171000181-434abc8e-19e1-48d3-a7e1-a4ca744cd5a0.png)

**Reply by imtheaman** [reply]
when pressed R i got --recording@r printed below. how should i use it?

**Reply by imtheaman** [reply]
got it. Thanks
